x-pack/plugins/lists/server/services/exception_lists/update_exception_list_item.ts

```diff
@@ -43,7 +43,7 @@ interface UpdateExceptionListItemOptions {
   user: string;
   tags: TagsOrUndefined;
   tieBreaker?: string;
-  type: ExceptionListTypeOrUndefined;
+  type: ExceptionListItemTypeOrUndefined;
 }
```

Comment:
Before both the list and item types were just `t.string` so it didn't scream at us, but now that the list type and item type differ, had to update.